packages/docs/docs/advanced/canvas-operations.mdx

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: 5
+slug: /canvas-operations
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+
+# Canvas Operations
+
+Because Motion Canvas is build ontop of the Browser's 2D Rendering Context, one
```

Comment:
```suggestion
Because Motion Canvas is build ontop of the Browser's 2D Rendering Context, we
```
On behalf of Hunter, 
https://github.com/motion-canvas/motion-canvas/pull/295#discussion_r1103335169

Reply:
Applied